spring-data-neo4j/src/main/java/org/springframework/data/neo4j/repository/query/Query.java

```diff
@@ -50,13 +51,6 @@ public Query(Filters filters) {
 		this.filters = filters;
 	}
 
-	public Query(String cypherQuery, Map<String, Object> parameters) {
-		Assert.notNull(cypherQuery, "Query must not be null.");
-		Assert.notNull(parameters, "Parameters must not be null.");
-		this.cypherQuery = sanitize(cypherQuery);
-		this.parameters = parameters;
-	}
-
 	public Query(String cypherQuery, String countQuery, Map<String, Object> parameters) {
 		Assert.notNull(cypherQuery, "Query must not be null.");
```

Comment:
Agree. I will return removed method for backward compatibility. 
And additionally  will add `Assert.notNull` check for `countQuery` to avoid `NullPointerException` in `sanitize`